include/random_walks/gaussian_cdhr_walk.hpp

```diff
@@ -9,6 +9,7 @@
 #define RANDOM_WALKS_GAUSSIAN_CDHR_WALK_HPP
 
 #include "generators/boost_random_number_generator.hpp"
+#include "sampling/sphere.hpp"
```

Comment:
this should go below `#include "random_walks/gaussian_helpers.hpp"` ?

Reply:
Yes it should be above `#include "random_walks/gaussian_helpers.hpp"`. Done